contrib/builder/deb/aarch64/generate.sh

```diff
@@ -83,12 +83,20 @@ for version in "${versions[@]}"; do
 			;;
 	esac
 
+    case "$suite" in
+        jessie)
+            echo 'RUN echo deb http://ftp.debian.org/debian jessie-backports main > /etc/apt/sources.list.d/backports.list' >> "$version/Dockerfile"
+            ;;
+        *)
+            ;;
+    esac
```

Comment:
Please help me out here. Stretch is currently in "testing" (according to https://wiki.debian.org/DebianStretch). Why exactly is this preferable over Jessie?